src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const states = [];
+  let currentState = { ...state };
```

Comment:
```suggestion
  const currentState = { ...state };
```
you will always work with this copy